### ISSUE TITLE
[FR] Use example content in code snippets by default

### ISSUE THREAD

**Issue by IanVS** [PaloAltoNetworks/docusaurus-openapi-docs#415]
### Is your feature request related to a problem?

Not so much a problem as something I think can be improved.  

### Describe the solution you'd like

It would be nice to show `example` data in the code snippets when an example is provided, for instance:

```
curl -L -X POST 'https://foo/bar' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
-H 'Authorization: Bearer <TOKEN>' \
--data-raw '{
  "name": "My new thing",
  "roleID": "role-PZEDBXHQEXKACJPZ6XOQTIAJA4"
}'
```

instead of 

```
curl -L -X POST 'https://foo/bar' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
-H 'Authorization: Bearer <TOKEN>' \
--data-raw '{
  "name": "string",
  "roleID": "string"
}'
```

Right now, to see the example, I have to click on the `Example` pill under `Body` in the demo panel.  It would be cool to default to the example, and only fall back to the schema-derived defaults when the example is missing.

### Additional context

I'm planning to hide the demo panel, because I don't want people to use it to hit our API (at least until we have a chance to set up a way to create a demo token or something.  So, the picker between the default and example won't be shown, but it would still be nice to show the example request body in the code snippets.

**Comment by IanVS** [comment]
Oh I just realized I can set the `example` property on each property itself, rather than an `example` on the schema, which I think accomplishes what I want.  But I think it's still maybe a good idea to default to the schema example.  Maybe.  Not sure, honestly.  Feel free to close this if you don't think it's a good idea.

**Comment by sserrata** [comment]
Thanks @IanVS, we're in the planning stages for a UI/UX overhaul/cleanup for v2.0.0. We'll definitely consider your suggestion to default to the first example.

I think the (first) example _should_ always be the preferred choice, but the downside is that it would be static, meaning it would require maintenance to keep it in sync with the schema.

**Comment by IanVS** [comment]
Great, thanks!  If you're in the planning stage, I hope you don't mind if I share a mockup that our designer created, which I'm able to _mostly_ implement with some CSS overrides, but I can't move the method+path block into the markdown (I don't think).  It would be pretty cool if that were supported in the new version.

<img width="929" alt="image" src="https://user-images.githubusercontent.com/4616705/218113178-f22ba00b-526d-4201-881c-d843beb85319.png">

vs what I've actually achieved so far:

<img width="1091" alt="image" src="https://user-images.githubusercontent.com/4616705/218113624-9f53c266-ab95-4bf1-8f5e-c6ce7a4df441.png">

**Comment by sserrata** [comment]
@blindaa121

**Comment by sserrata** [comment]
Hi @IanVS, would you mind sharing the CSS used to replace the details markers with the (IMO) cleaner arrows in your screenshot? Thanks!

**Comment by IanVS** [comment]
Sure, the site is live at https://docs.defined.net/API/delete-host/ if you want to inspect it there.  

This is the CSS I hacked together:

```css
[class*='mimeSchemaContainer_'] summary::before {
  content: '';
  background: var(--ifm-menu-link-sublist-icon) 50% / 1.2rem 1.2rem;
  filter: var(--ifm-menu-link-sublist-icon-filter);
  height: 0.75rem;
  transform: rotate(90deg) !important;
  width: 0.75rem;
  transition: transform var(--ifm-transition-fast) linear !important;
  border: none !important;
  transform-origin: unset !important;
}

[class*='mimeSchemaContainer_'] details[data-collapsed='false'] > summary::before {
  transform: rotate(180deg) !important;
}
```

**Comment by sserrata** [comment]
Thanks! Here's a variation that should cover all the markers in the first/left-hand column:

```
.theme-api-markdown > div:first-child summary::before {
  content: "";
  background: var(--ifm-menu-link-sublist-icon) 50% / 1.2rem 1.2rem;
  filter: var(--ifm-menu-link-sublist-icon-filter);
  height: 0.75rem;
  transform: rotate(90deg) !important;
  width: 0.75rem;
  transition: transform var(--ifm-transition-fast) linear !important;
  border: none !important;
  transform-origin: unset !important;
}

.theme-api-markdown
  > div:first-child
  details[data-collapsed="false"]
  > summary::before {
  transform: rotate(180deg) !important;
}
```

**Comment by cryptodam** [comment]
Hello @IanVS ,
I read your thread but I did not manage to put examples of value (like `"my new host" `in the code snipets as you managed to do in your example above. 
I tried to add example at the parameter level in my schema description but seems it s ignoring it. Did you activate something?
<img width="268" alt="Screenshot 2023-06-05 at 23 12 04" src="https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/assets/126563094/5d309801-9153-4419-9f5f-2455edd16890">
Thanks in advance for your help
